### ISSUE TITLE
Wondering if we want the option to use use _draw() 

### ISSUE THREAD

**Issue by jtaart** [arypbatista/godot-bordered-polygon-2d#23]
I have the function pretty much built, but not sure if we need it. It might be nice to have the option if someone does not care about materials.

**Comment by arypbatista** [comment]
Yeah, I will make this a enhancement. Let's put it on hold until someone ask for it or we have the need to do it.

**Comment by Keyaku** [comment]
Do you mean using the `draw_*` functions from `CanvasItem`?  
If so, if borders are to optionally have collision in them (as per #42), this is not a good idea.

**Comment by arypbatista** [comment]
@Keyaku the drawing approach can defer to the collision building. We can draw by using the CanvasItem functions or by creating more Polygon2D to complete borders. Whatever we choose we will be spawning a StaticBody2D with a CollisionPolygon2D that covers all the surface of the BorderedPolygon.

Makes sense?

**Comment by Keyaku** [comment]
Ah yes, my mistake. The polygon data is not lost in the process, so it can be applied to a CollisionPolygon2D.

**Comment by arypbatista** [comment]
Still, this is a good discussion. I don't exactly know how node count affects performance here. That is something we will need to optimize at some point.
